protocol/communities/manager.go

```diff
@@ -1310,12 +1417,42 @@ func (m *Manager) DeleteCategory(request *requests.DeleteCommunityCategory) (*Co
 	return changes.Community, changes, nil
 }
 
-func (m *Manager) HandleCommunityDescriptionMessage(signer *ecdsa.PublicKey, description *protobuf.CommunityDescription, payload []byte) (*CommunityResponse, error) {
+func (m *Manager) Queue(signer *ecdsa.PublicKey, community *Community, description *protobuf.CommunityDescription, payload []byte) error {
+
+	m.logger.Info("queuing community", zap.String("id", community.IDString()), zap.String("signer", common.PubkeyToHex(signer)))
+
+	communityToValidate := communityToValidate{
+		id:         community.ID(),
+		clock:      community.config.CommunityDescription.Clock,
```

Comment:
Not clear which role the clock is playing here. I do not see it being used in the run loop, also this is a previous community clock and not a new one

Reply:
We pull them from the database in reverse clock order, and as soon as one validates, we stop validating any other with the same id, good point about the clock, I'll fix it :)